src/main/java/unrc/dose/Comment.java

```diff
@@ -8,10 +12,52 @@
 * description     :varchar(300)    not null
 * challenge_id    :integer     not null
 * user_id         :integer    not null
+* comment_id      :integer
 **/
-package unrc.dose;
+/**
+*@author Celi Yanina, de Prada Hernan, Goldenberg Erika
+**/
+public class Comment extends Model {
 
-import org.javalite.activejdbc.Model;
+  /**
+  *Create a new comment.
+  *@param title this is the comment's title
+  *@param description this the comment's body
+  *@param challenge_id this is the challenge's id which is commented
+  *@param user_id this is the user's id who commented
+  **/
 
-public class Comment extends Model {
+  public void createComment (String title,String description, int challenge_id, int user_id ){
+    this.set("title",title);
+    this.set("description",description);
+    this.set("challenge_id", challenge_id);
+    this.set("user_id", user_id);
+    this.saveIt();
+  }
+  /*
+  *isResponse
+  *@param comment_id this is comments or response to evaluate
+  *@return return if the selected comment_id si a comment or a response.
+  **/
+  public boolean isResponse(int comment_id){
+    Comment c = Comment.findById(comment_id);
+    return ((c.getInteger("comment_id")) != (null));
```

Comment:
Please be careful since the code will give false positives, consider this case:

There are two comments in the database, `comment1`, `comment2`, and `comment3` where the second is a reply to the first one, and the third one is just a new comment which is not replying to any other comment.

Consider that one is calling `c.isResponse(1)` where `c` correspond to `comment3`, since there is a comment with id `3` this method will return `true` but neither `comment1` nor `comment3` is a response.

This method should not take any argument and the code should use `getInteger("comment_id") != null`